docker/centos/Dockerfile

```diff
@@ -35,7 +35,7 @@ ENV MINIFI_BASE_DIR /opt/minifi
 RUN mkdir -p $MINIFI_BASE_DIR 
 USER $USER
 
-RUN yum -y install java-1.8.0-openjdk java-1.8.0-openjdk-devel sudo git which maven
+RUN yum -y install java-1.8.0-openjdk java-1.8.0-openjdk-devel gcc g++ sudo git which maven
```

Comment:
not entirely necessary as bootstrap will install it but this is easier to see in the output.